cartridges/app_storefront_pdp_extend/cartridge/scripts/helpers/notificationHelpers.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+function getClientContentAsset(contentAssetID) {
+    var ContentMgr = require('dw/content/ContentMgr');
+    var customerContent = ContentMgr.getContent(contentAssetID);
+    return customerContent;
+}
+
+function getCustomObject(customObjectType, primaryKey) {
```

Comment:
This one and the one below are generic for COs hence can go into a new helper customObjectsHelper